### PR TITLE
fix send button not overlapping completely and not returning to origi…

### PR DESCRIPTION
**Pull request by LanderN**
…nal state after delayed send

The bars of the button in the "send" state did not overlap completely at the tip and after a delayed message, the bars did not return to the + sign, but to a strange T-shape.
This commit fixes those issues.

### CODE REVIEW

File: QKSMS/src/main/java/com/moez/QKSMS/ui/view/ComposeView.java [hunk 1]
```diff
@@ -367,23 +367,19 @@ private void updateButtonState(SendButtonState buttonState) {
             float barRotationTarget2 = mButtonState == SendButtonState.ATTACH ? 90 : 135;
             float barTranslationTarget2 = mButtonState == SendButtonState.SEND ? translation : 0;
 
-            if (barRotation1 != barRotationTarget1) {
                 ObjectAnimator.ofFloat(mButtonBar1, "rotation", barRotation1, barRotationTarget1)
                         .setDuration(ANIMATION_DURATION)
                         .start();
                 ObjectAnimator.ofFloat(mButtonBar2, "rotation", barRotation2, barRotationTarget2)
                         .setDuration(ANIMATION_DURATION)
                         .start();
```
Comment (moezbhatti) [inline QKSMS/src/main/java/com/moez/QKSMS/ui/view/ComposeView.java]: Now that the code isn't nested, they're wrongly indented

Can you remove the extra indentation?

**Comment by moezbhatti** [comment]
Cool, thanks. Will test this when I get home

**Comment by moezbhatti** [comment]
Sweet, this fixes it

**Comment by moezbhatti** [comment]
Can you rebase your branch off of the latest version of develop?

**Comment by LanderN** [comment]
Hi, I'm sorry for my ignorance, but could you tell me how to do this? 
I tried "git rebase qklabs/develop" but it doesn't seem to work?

**Comment by LanderN** [comment]
I fixed it (I think?), I'll try not to make a mess next time O:)

**Pull request closed without merging**